### ISSUE TITLE
feature discussion

### ISSUE THREAD

**Issue by MarsVard** [UFreedom/FloatingText#5]
It seems to me that it would not be too hard to implement a feature to allow you to float any view(Imageview, textview, ...)

Is this something you want to cover in this library or would you rather keep only compatible with textview?

**Comment by UFreedom** [comment]
In fact, I have this idea, reference the implementations  of   FloatingTextViewdeas,We can float any effect View, How about you, Do you have a detailed ideas about it?

**Comment by MarsVard** [comment]
I think it would be nice to be able to pass any view to the FloatingText. what do you think?

Something like this:
textView = new TextView(activity);
textView.setText("SWAG");
textView.setTextColor(...)

floatingText = new FloatingText.FloatingTextBuilder(activity)
                               .offsetX(100) // the x offset  relate to the attached view
                               .offsetY(100) // the y offset  relate to the attached view  
                               .floatingAnimatorEffect(FloatingAnimator) // floating animation
                               .floatingPathEffect(FloatingPathEffect) // floating path
                               .build();

floatingText.float(textView);

**Comment by UFreedom** [comment]
Yes, your idea is great!, But  the name "FloatTextView"  sounds just like floating Text,How about use other name such as  FloatingEffect ,FloatingView.or we can create a new project, and then implement it together.

Are you from China?

**Comment by MarsVard** [comment]
I think we can just fork this and call it FloatingView?

I'm from Belgium.

**Comment by UFreedom** [comment]
@MarsVard 
I'm sorry so long to get back to you.
As I know,In github,we can't fork our own repo,But i have find this post https://kroltech.com/2014/01/01/quick-tip-how-to-fork-your-own-repo-in-github/. So i follow this tutorial and create a new repo FloatingView ,here is : https://github.com/UFreedom/FloatingView.

I'm not sure the repo  FloatingView is that your mean "I think we can just fork this and call it FloatingView?" ?

**Comment by MarsVard** [comment]
Hey @UFreedom That's exactly what I meant. But I have been busy myself I'll try to make some time to see what can be done on that repo this weekend.

**Comment by UFreedom** [comment]
@MarsVard 
that's ok,I am also busy ,and we just do our best to make it better。

Thanks,

**Comment by UFreedom** [comment]
@MarsVard 
I have finished the [FloatingView](https://github.com/UFreedom/FloatingView) develop.

**Comment by MarsVard** [comment]
@UFreedom Thanks for letting me know! I have been so busy I totally forgot about this project.

It looks awesome!
